src/NexusMods.MnemonicDB.Storage/DatomStore.cs

```diff
@@ -268,7 +287,8 @@ private async Task Bootstrap()
         }
 
         _updatesSubject = new BehaviorSubject<(TxId TxId, ISnapshot snapshot)>((_asOfTx, _currentSnapshot));
-        _txTask = Task.Run(ConsumeTransactions);
+        _loggerThread = new Thread(ConsumeTransactions);
+        _loggerThread.Start();
```

Comment:
I don't think this thread is ever stopped, should we add a cancellation token and cancel it on Dispose?

I noticed that the tests kept running after completion and needed to be manually killed. I think this might be the cause but could also be something else.

Reply:
We can make this more robust, but the thread ends when the queue completes, and the queue completes when the object is disposed